client/landing/jetpack-cloud/components/daily-backup-status/index.jsx

```diff
@@ -283,9 +297,16 @@ class DailyBackupStatus extends Component {
 
 		return (
 			<>
-				<Gridicon className="daily-backup-status__gridicon-backup-scheduled" icon="cloud-upload" />
-				<div className="daily-backup-status__static-title">
-					{ translate( 'Backup Scheduled:' ) }
+				<div className="daily-backup-status__message-head">
+					<img src={ cloudScheduleIcon } alt="" role="presentation" />
+					<div className="daily-backup-status__hide-mobile">
+						{ translate( 'Backup Scheduled' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.